docs/index.rst

```diff
@@ -89,9 +89,13 @@ Understand messages, hold conversations, and connect to messaging channels and A
    migration-guide
    changelog
 
-.. toctree:
+.. toctree::
    :maxdepth: 1
    :hidden:
-   :caption: Migration (beta)
+   :caption: Migrate from (beta)
```

Comment:
just wondering, why is this always beta? Because the others' formats could change and then suddenly our scripts don't work?

Reply:
that's part of it. Also we've never really made the migration feature-complete (composite entities are a recent example). It's labelled as beta to communicate that you can get up and running quickly, but it won't be a like-for-like port of your existing app